WebRoot/js/zimbraMail/core/ZmOperation.js

```diff
@@ -306,7 +306,7 @@ ZmOperation.defineOperation =
 function(baseId, overrides) {
 	var id = (overrides && overrides.id) || (baseId && baseId.id) || baseId || Dwt.getNextId();
 	var textKey = (overrides && overrides.textKey) || ZmOperation.getProp(baseId, "textKey");
-	var text = textKey && ZmMsg[textKey];
+	var text = textKey && ZmMsg[textKey] || textKey;
 	var tooltipKey = (overrides && overrides.tooltipKey) || ZmOperation.getProp(baseId, "tooltipKey");
 	var tooltip = tooltipKey && ZmMsg[tooltipKey];
```

Comment:
same fix for tooltip, although we are not using it but for consistency

Reply:
Done.